tests/utils/worker_utils.cpp

```diff
@@ -1,15 +1,17 @@
+#include "faabric/util/config.h"
```

Comment:
I've tried to make this file more high-level and do less interrogation of the Faabric internals, hence there's a lot of red.